### DISCUSSION TITLE
unarchive followed by sql_insert

### DISCUSSION THREAD

**Discussion by rdilipk** [redpanda-data/connect#1165] [Q&A] [answered]
Am I right in understanding that if I run a message through the `unarchive` processor and my structured document gets split into individual messages, each message is independently delivered to the output processor? IOW, if I have a `sql_insert` output processor, n individual messages will call sql_insert n times?

**Comment by Jeffail** [comment, marked as answer]
hey @rdilipk yeah that's the case
